### PR TITLE
feat(jetbrains-gateway): add releases_base_link/download_base_link variables

### PR DESCRIPTION
**Pull request by kerwin612**
* Enhance the jetbrains-gateway module to support custom download link.

In [offline mode](https://www.jetbrains.com/help/idea/fully-offline-mode.html), the currently configured offline download address will be overridden by the parameters of jetbrains-gateway(jetbrains-gateway://connect#......&ide_download_link=https://download.jetbrains.com/idea/ideaIU-XXX.tar.gz).

### CODE REVIEW

**Review by matifali — APPROVED** [review]
LGTM. Thank you for the contribution. 
Could you also add some instructions in README.md on how one can set up custom jetbrains hosting?

File: jetbrains-gateway/README.md [hunk 1]
```diff
@@ -72,6 +72,22 @@ module "jetbrains_gateway" {
...
+  agent_id              = coder_agent.example.id
+  agent_name            = "example"
+  folder                = "/home/coder/example"
+  jetbrains_ides        = ["GO", "WS"]
+  releases_base_link    = "https://releases.internal.site/"
+  download_base_link    = "https://download.internal.site/"
+  default               = "GO"
+}
+```
+
```
Comment (matifali) [inline jetbrains-gateway/README.md:90]: Thank you. This example is helpful. But can you also add on how one can set up internal serving for JetBrains IDEs? Thanks

**Review by matifali — APPROVED** [review]

**Review by matifali — APPROVED** [review]

**Comment by matifali** [comment]
Hi @kerwin612, could you run `bun install && bun update-version` to bump the module version?
Thanks.

**Comment by kerwin612** [comment]
> Hi @kerwin612, could you run `bun install && bun update-version` to bump the module version? Thanks.
The feedback after running is as follows, but I'm not quite sure what caused it.
```bash
bun update-version
$ ./update-version.sh
fatal: No names found, cannot describe anything.
error: script "update-version" exited with code 128
```

**Pull request merged**